### PR TITLE
Add documentation for document type

### PR DESCRIPTION
**Pull request by SISheogorath**
Looks like we have a document type slide out there in the wild which is completely undocumented but really awesome.

So let's make it know to the people, so they can make use of it!

### CODE REVIEW

File: public/docs/yaml-metadata.md [hunk 1]
```diff
@@ -75,21 +75,21 @@ https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes
...
 **Example**
-```xml
+```yml
 langs: ja-jp
 ```
 
 dir
 ---
-This option provide to describe the direction of the text in this note.
+This option provides to describe the direction of the text in this note.
```
Comment (ccoenen) [inline public/docs/yaml-metadata.md]: -> "This option specifies the direction of the text in this note."

File: public/docs/yaml-metadata.md [hunk 2]
```diff
@@ -102,35 +102,46 @@ You can only use whether `true` or `false`.
...
 
 **Example**
-```xml
+```yml
 disqus: codimd
 ```
 
+type
+---
+This option allows you to switch the view to the slide mode.
```
Comment (ccoenen) [inline public/docs/yaml-metadata.md]: "view to slide mode" is not very clear. This sounds like activating presentation mode. My own attempts at making this more clear failed, though.

File: public/docs/yaml-metadata.md [hunk 3]
```diff
@@ -102,35 +102,46 @@ You can only use whether `true` or `false`.
...
+
+**Example:**
+```yml
+type: slide
+```
+
 slideOptions
 ---
-This option allow you provide custom options to slide mode.
+This option allows you provide custom options to slide mode.
```
Comment (ccoenen) [inline public/docs/yaml-metadata.md]: -> "... allows you TO provide ..."

**Comment by SISheogorath** [comment]
@ccoenen please have another look

**Pull request merged**